input/intro-notes/StructureDefinition-ehealth-questionnaire-intro.md

```diff
@@ -55,4 +55,13 @@ The `helpText.xhtml` value should follow the same standards as http://hl7.org/fh
 The `item.enableBehavior.conditionId` can be used to identify conditions. This property is primarily used when modeling Questionnaires which should be transformed to CDA documents.
 
 ### External identifier
-The `item.externalIdentifier` can be used to identify either questionnaire groups or questionnaire questions. This property is primarily used when modeling Questionnaires which should be transformed to CDA documents.
\ No newline at end of file
+The `item.externalIdentifier` can be used to identify either questionnaire groups or questionnaire questions. This property is primarily used when modeling Questionnaires which should be transformed to CDA documents.
+
+### Copyright
+The `item.isCopyright` can be used to indicate a copyright structure.
+`item.isCopyright` is a boolean value, and if set to true, the item must have either `item.type`: 'group' or 'display'. Furthermore, for `item.type` 'group' the isCopyright is only allowed on root items, for items with type 'display' the `item.isCopyright` is only allowed true if the parent item is of type 'group', isCopyright equals true and the group item is on root level.
```

Comment:
`item.isCopyright` is a boolean value, and if set to true, the item must have either `item.type`: 'group' or 'display'. Furthermore, for `item.type` 'group', the isCopyright is only allowed on root items. 
For items with type 'display', the `item.isCopyright` is only allowed true if:
- the parent item is of type 'group'
- isCopyright equals true
- the group item is on root level